docs/common-options.md

```diff
@@ -0,0 +1,140 @@
+
+The `fieldsObj` option is an `Object` who properties can be any [analytics.js field name](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference), and whose values will be used as the corresponding field value for all hits sent by the plugin.
+
+### Examples
+
+#### `mediaQueryTracker`
+
+This configuration ensures all events sent by the `mediaQueryTracker` plugin are non-interaction events:
+
+```js
+ga('require', 'mediaQueryTracker', {
+  definitions: [...],
+  fieldsObj: {
+    nonInteraction: true
+  }
+});
+```
+
+#### `urlChangeTracker`
+
+This configuration sets a [custom dimension](https://support.google.com/analytics/answer/2709828) at index 1 for all pageview hits sent by the `urlChangeTracker`. This would allow you to differentiate between the initial pageview and "virtual" pageviews sent after loading new pages via AJAX:
+
+```js
+ga('require', 'urlChangeTracker', {
+  fieldsObj: {
+    dimension1: 'virtual'
+  }
+});
+ga('send', 'pageview', {
+  dimension1: 'pageload'
+});
+```
+
+## `attributePrefix`
+
+All plugins that send hits to Google Analytics as a result of user interactions with DOM elements support declarative attribute binding (see the [`eventTracker`](/docs/plugins/event-tracker.md) plugin to see how this works). As such, each of these plugins accept an `attributePrefix` option to customize what attribute prefix to use.
+
+By default, the `attributePrefix` value used by each plugin is the string `'ga-'`, though that value can be customized on a per-plugin basis.
+
+**Note:** when setting the same field in both the `fieldsObj` option as well as via a DOM element attribute, the attribute's value will override the `fieldsObj` value.
+
+### Examples
+
+#### `eventTracker`
+
+```js
+ga('require', 'eventTracker', {
+  attributePrefix: 'data-'
+});
+```
+
+```html
+<button
+  data-on="click"
+  data-event-category="Video"
+  data-event-action="play">
+  Play video
+</button>
+```
+
+#### `impressionTracker`
+
+```js
+ga('require', 'impressionTracker', {
+  elements: ['cta'],
+  attributePrefix: 'data-ga'
+});
+```
+
+```html
+<div
+  id="cta"
+  data-ga-event-category="Call to action"
+  data-ga-event-action="seen">
+  Call to action
+</a>
+```
+
+#### `outboundLinkTracker`
+
+```js
+ga('require', 'outboundLinkTracker', {
+  attributePrefix: ''
+});
+```
+
+```html
+<a href="https://example.com" event-category="External Link">Click</a>
+```
+
+## `hitFilter`
+
+The `hitFilter` option is useful when you need to make more advanced modifications to a hit, or when you need to abort the hit altogether. `hitFilter` is a function that gets invoked with the tracker's [model object](https://developers.google.com/analytics/devguides/collection/analyticsjs/model-object-reference) as its first argument, and, if the hit was initiated from a user interaction with a DOM element, the element as the second argument.
```

Comment:
suggestion for last part of sentence:
...as its first argument and the element as the second argument if the hit was initiated from a user interaction with a DOM element.